packages/story-editor/src/components/panels/document/taxonomies/HierarchicalTermSelector.js

```diff
@@ -131,14 +207,24 @@ function HierarchicalTermSelector({ taxonomy }) {
             id={dropdownId}
             ariaLabel={taxonomy.labels.parent_item}
             placeholder={taxonomy.labels.parent_item}
-            options={[]}
+            options={categories}
+            selectedValue={selectedParent}
+            onMenuItemClick={handleParentSelect}
```

Comment:
Might be nice to add a submission event if the user presses the  `Enter` key here while this is focused

Reply:
Adding another issue for this: Hitting `Enter` works EXCEPT when the dropdown is here. Guessing the dropdown is grabbing the enter event for some reason.

We can look into this further so that the `Enter` key submits rather than focusing the dropdown. https://github.com/google/web-stories-wp/issues/9059